src/client/driver/driver.js

```diff
@@ -1110,6 +1162,18 @@ export default class Driver extends serviceUtils.EventEmitter {
         }
     }
 
+    async _restoreChildWindowLinks () {
+        if (!this.contextStorage.getItem(PENDING_CHILD_WINDOW_COUNT))
+            return Promise.resolve();
+
+        return getTimeLimitedPromise(new Promise(resolve => {
+            this._restoreChildWindowsPromiseResolver = resolve;
+        }), RESTORE_CHILD_WINDOWS_TIMEOUT)
+            .catch(() => {
```

Comment:
Remove dev artifacts.

Reply:
this is not a dev artifact.
we need an empty line inside the catch block according to our linter rules.
As for the catch block. I decided not to throw an error if the link is not restored. I throw the error that the window is not found.

Reply:
It's better to raise a specific error ('Failed to restore connection to window within the allocated timeout').
It will allow to localize the problematic code place on errors.

Reply:
Also let's rewrite the `_restoreChildWindowLinks` using `async/await` statements.